src/main/java/io/github/bobocodebreskul/context/support/BeanDefinitionReaderUtils.java

```diff
@@ -73,7 +87,8 @@ public static List<Class<?>> getBeanDependencies(Class<?> beanClass) {
 
     return Stream.concat(Stream.concat(constructorDependenciesStream, fieldDependenciesStream),
             methodDependenciesStream)
-        .collect(Collectors.toList());
+        .map(dependency -> new BeanDependency(generateClassBeanName(dependency), dependency))
```

Comment:
may it broke something if we remove other dependencies except constractor from concat method? Because in case if add @Autowired somewhere it looks like it added additional dependencies which will broke logic from Container

Reply:
I'll do it in scope of this task https://github.com/bobocode-breskul/bring/issues/66